src/praisonai/praisonai/integrations/compute_managed_agent.py

```
@@ -0,0 +1,257 @@
+    def is_available(self) -> bool:
+        try:
+            return bool(getattr(self._resolve(), "is_available", True))
+        except Exception:
+            return False
+
+    # ── ManagedBackendProtocol ───────────────────────────────────────────────
+    async def execute(self, prompt: str, **kwargs) -> str:
+        await self._ensure()
+
+        payload = json.dumps({"agent": self._config, "prompt": prompt})
+        await self._write("/tmp/praison_agent.json", payload)
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/praison_agent.json"

(S108)